images/manageiq-base/Dockerfile

```diff
@@ -1,14 +1,22 @@
-FROM registry.access.redhat.com/ubi9/ubi
-MAINTAINER ManageIQ https://manageiq.org
+ARG FROM_REPO=docker.io/manageiq
+ARG FROM_TAG=latest
+
+# alias stage for rpms
+FROM ${FROM_REPO}/manageiq-rpms:${FROM_TAG} as rpms
```

Comment:
docker cannot handle ARG interpolation in `RUN --mount=type=bind,from=`.  When you do this you get an error and it tells you to create an alias stage instead, which is what this is.